package/utils/yafut/Makefile

```
@@ -0,0 +1,35 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=yafut
+PKG_RELEASE:=$(AUTORELEASE)
```

I _think_ that OpenWrt was looking to move away from `AUTORELEASE`. There is a check in the packages branch against it: https://github.com/openwrt/packages/blob/master/.github/workflows/check-autorelease-deprecation.yml
```suggestion
PKG_RELEASE:=1
```

---

Ah, of course, thanks, I wasn't aware of this. Will fix.

---

Addressed as of ca52c328051a02888e18354be2d58f1ff96ff738, thanks.